.github/workflows/build_and_test.yml

```diff
@@ -285,6 +285,8 @@ jobs:
   lint:
     name: Linters, licenses, dependencies and documentation generation
     runs-on: ubuntu-20.04
+    container:
+      image: dongjoon/apache-spark-github-action-image:20201025
```

Comment:
BTW, we might need to have a way to control the dependency in this image soon ...